content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,95 @@ title: Custom Metrics
 kind: documentation
 ---
 
-If your Lambda functions are sending trace or log data to Datadog, and the data you want to query is captured in an existing log or trace, you can [generate custom metrics from logs and traces](#creating-custom-metrics-from-logs-or-traces) without re-deploying or making any changes to your application code. If you want to send new data not captured in an existing log or trace, you can [send custom metrics using the Datadog Lambda Library](#sending-custom-metrics-from-the-datadog-lambda-library).
+## Overview
+
+There are a few different ways to submit custom metrics to Datadog from a Lambda function. 
+
+- **Creating custom metrics from logs or traces**: If your Lambda functions are already sending trace or log data to Datadog, and the data you want to query is captured in an existing log or trace, you can [generate custom metrics from logs and traces](#creating-custom-metrics-from-logs-or-traces) without re-deploying or making any changes to your application code.
+- **Submitting custom metrics synchronously**: If you want to generate custom metrics from a non-performance-critical Lambda function written in Python, Node.js or Golang and you are not interested in collecting traces, you can use Datadog Lambda Library to [submit custom metrics synchronously](#enabling-synchronous-custom-metrics).
```

Comment:
Should we be encouraging this option? Isn't the extension strictly better for this use case? I think it would be nice to eventually remove this feature.

Reply:
At the very least, I think this should be listed below the asynchronous option, as that would be preferable for most customers.

Reply:
We can put it a bit lower on the list with a (Not recommended) in front

Reply:
That's good feedback, actually I think we can remove this solution from the doc, the gap leaving behind seems minimal.